I'm sorry, but an uncaught exception occurred.

While loading <renpy.display.im.Image object ('gui/scrollbar/horizontal_[prefix_]thumb.png') at 0x000000000550d550>:
  File "game/minigames/game/script.rpy", line 6, in merge_hi_scores
    current.update(old)
TypeError: 'NoneType' object is not iterable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "E:\новела\renpy-8.1.3-sdk\renpy\bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "E:\новела\renpy-8.1.3-sdk\renpy\main.py", line 684, in main
    renpy.persistent.update(True)
  File "E:\новела\renpy-8.1.3-sdk\renpy\persistent.py", line 396, in update
    merge(other)
  File "E:\новела\renpy-8.1.3-sdk\renpy\persistent.py", line 341, in merge
    val = merge_func(old, new, pval)
  File "game/minigames/game/script.rpy", line 6, in merge_hi_scores
    current.update(old)
  File "E:\новела\renpy-8.1.3-sdk\renpy\revertable.py", line 97, in do_mutation
    return method(self, *args, **kwargs)
TypeError: 'NoneType' object is not iterable

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Путь Программиста 1.0
Tue Oct 10 11:01:16 2023
